charts/k8tz/values.yaml

```diff
@@ -10,6 +10,9 @@ injectAll: true
 cronJobTimeZone: false  # requires kubernetes >=1.24.0-beta.0 with 'CronJobTimeZone' feature gate enabled (alpha)
 verbose: false
 
+# Labels to apply to all resources
+commonLabels: {}
```

Comment:
changes to the chart values must be documented in the [chart readme](https://github.com/k8tz/k8tz/blob/master/charts/k8tz/README.md#values)